java/connector-server-zip/pom.xml

```diff
@@ -85,6 +85,13 @@
       <groupId>ch.qos.logback</groupId>
       <artifactId>logback-classic</artifactId>
     </dependency>
+
+    <!-- JUL to SLF4J bridge-->
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>jul-to-slf4j</artifactId>
+      <version>${slf4j.version}</version>
```

Comment:
You should remove `<version/>` here

Reply:
Thanks, yes this is indeed redundant, I will remove it.